src/Symfony/Component/PropertyAccess/PropertyAccessorBuilder.php

```diff
@@ -53,6 +54,30 @@ public function disableMagicCall()
         return $this;
     }
 
+    /**
+     * Enables the use of static properties/methods by the PropertyAccessor.
```

Comment:
I think it's useless

Reply:
I reproduce same as `magicCall` option, we have both enable / disable methods for this one too, but since staticCall is enabled by default I can remove the enable method ?

Reply:
I was talking about the PHPDoc